CHANGELOG.md

```diff
@@ -15,6 +15,15 @@
 
 ### [3.3.1](https://www.github.com/googleapis/java-spanner/compare/v3.3.0...v3.3.1) (2021-01-14)
 
+### ⚠ IMPORTANT: Known issue with this version of the client
+
+Since [v3.0.0](https://github.com/googleapis/java-spanner/releases/tag/v3.0.0), users might experience transaction getting stuck in a very rare occasion, where the Spanner backend returns a retryable error when consuming the first record of a read / query in a transaction.
```

Comment:
"users might experience transaction getting stuck in a very rare occasion, where" --> transactions can get stuck if

Reply:
Done.